scripts/test.sh

```diff
@@ -16,7 +16,7 @@ echo "Packages to test: $PACKAGES"
 find . -name 'cprofile*.out' -exec sh -c 'rm "{}"' \;
 find . -type d -name data -not -path './vendor/*' -not -path './data' | xargs rm -rf
 
-$GO test $GOFLAGS -run=$TESTS $TESTFLAGS -v -timeout=2000s -covermode=count -coverpkg=$PACKAGES_COMMA -exec $DIR/test-xprog.sh $PACKAGES 2>&1 > >( tee output )
+$GO test $GOFLAGS -run=$TESTS $TESTFLAGS -v -timeout=1000s -covermode=count -coverpkg=$PACKAGES_COMMA -exec $DIR/test-xprog.sh $PACKAGES 2>&1 > >( tee output )
```

Comment:
- Is 1000s enough for CI tests these days?
- Should we switch to a more human friendly duration format? e.g. 20m

Reply:
Oh my bad, it should be 2000s. I'll change it to 20m

Reply:
Cool. Maybe 20m wasn't the best example as it's closer to 1000s than 2000s but quickly looking at CI times it seems that the average is around 15-16 minutes so it should be okay. I guess we could always could bump that to half an hour to be extra-safe.